src/domain/interfaces/locking-api.interface.ts

```diff
@@ -20,6 +21,12 @@ export interface ILockingApi {
     offset?: number;
   }): Promise<Page<Rank>>;
 
+  getLeaderboardV2(args: {
```

Comment:
Note that the Safe Locking Service implementation changed recently, and the leaderboard by campaign endpoint is not under the `v2` prefix anymore. These changes can be seen here (approval is pending): https://github.com/safe-global/safe-locking-service/pull/100/files#diff-d790dada13951d227666cb46aa0ccc89033037f7a7a5e1ed86e9bbfbbe0ef65b

A follow-up PR will be created to modify the datasource endpoints and rename functions/variables accordingly.